app/polymer/src/server/framework-preset-polymer.ts

```diff
@@ -10,7 +10,7 @@ export function webpack(config) {
         {
           test: /\.html$/,
           use: [
-            ...config.module.rules[0].use,
+            ...(config.module.rules[0].use as RuleSetUseItem[]),
```

Comment:
tslint complained about the typing of `use`. I had to cast it but it might be a bug

`rules[0].use` can be ```    type RuleSetUse =
        | RuleSetUseItem
        | RuleSetUseItem[]
        | ((data: any) => RuleSetUseItem | RuleSetUseItem[]);```

or are we 100% certain it is always `RuleSetUseItem[]`? Do we need to add a check here?

Reply:
We can't be not sure, we should probably check